generators/base.py

```diff
@@ -1,6 +1,14 @@
 import abc
 from typing import Any, Dict
 
+class BaseIssueGenerator(abc.ABC):
+    @abc.abstractmethod
+    def generate_issue(self, scorecard: Dict[str, Any], entity: Dict[str, Any], blueprint: str, level: str, tasks: list[str]):
+        pass
+
+    @abc.abstractmethod
+    def generate_task(self, rule: Dict[str, Any]) -> str:
+        pass
```

Comment:
I feel like you could've used the `BaseTicketGenerator` let me know what you think

Reply:
I used a different one to have consistent naming (task vs issue etc)
Think I should switch to the  `BaseTicketGenerator` ?